src/uw_basic.jl

```diff
@@ -123,7 +123,15 @@ $(SIGNATURES)
 Compute resonance frequency of a freely oscillating has bubble in water, given:
 - bubble `radius` in meters
 - `depth` of bubble in water in meters
+- γ : gas ratio of specific heats. Default 1.4 for air 
```

Comment:
Please follow same doc-string style as other parameters for consistency

Reply:
Will do. Also will remove the default values from the docstring. But isnt it good to mention the default is air, somewhere?

Reply:
I'm fine with mentioning defaults. Use format "(default: xxx)" at the end of the parameter description.